docs/use/affinity.md

```diff
@@ -0,0 +1,170 @@
+#  Leveraging Affinity in Percona Everest
+
+!!! warning
```

Comment:
Maybe it should just be an info or note instead of a warning

Reply:
@recharte Let's say it is Important.